### PR TITLE
LOOP-1488: add delivery limits information screen + editors

### PR DESCRIPTION
**Pull request by novalegra**
https://tidepool.atlassian.net/browse/LOOP-1488

### CODE REVIEW

File: Loop/Managers/ServicesManager.swift [hunk 1]
```diff
@@ -23,6 +23,8 @@ class ServicesManager {
 
     private let servicesLock = UnfairLock()
 
+    weak var dataManager: LoopDataManager?
```
Comment (mpangburn) [inline Loop/Managers/ServicesManager.swift]: Curious—what's this addition for?

File: Loop/Models/LoopSettings+Loop.swift [hunk 2]
```diff
@@ -7,6 +7,7 @@
 
 import Foundation
 import LoopCore
+import LoopKit
```
Comment (mpangburn) [inline Loop/Models/LoopSettings+Loop.swift]: What makes this import required now?

**Review by mpangburn — APPROVED** [review]
Questions re changes in files that seem unrelated, but otherwise LGTM

File: Loop/Models/LoopSettings+Loop.swift (hunk 2 above)
Comment (novalegra) [reply Loop/Models/LoopSettings+Loop.swift]: Not required, it got removed in a prior PR but I forgot to remove it in this one. Will fix

File: Loop/Managers/ServicesManager.swift (hunk 1 above)
Comment (novalegra) [reply Loop/Managers/ServicesManager.swift]: This is from me not noticing that while resolving merge conflicts, removing! Thanks for catching that

File: Loop/Managers/LoopDataManager.swift [hunk 3]
```diff
@@ -1826,6 +1833,7 @@ extension LoopDataManager {
                         maximumBolus: settings.maximumBolus,
                         suspendThreshold: settings.suspendThreshold,
                         insulinSensitivitySchedule: insulinSensitivitySchedule,
-                        carbRatioSchedule: carbRatioSchedule)
+                        carbRatioSchedule: carbRatioSchedule,
+                        glucoseUnit: glucoseUnit)
```
Comment (rickpasetto) [inline Loop/Managers/LoopDataManager.swift]: See my [comment](https://github.com/tidepool-org/LoopKit/pull/142#discussion_r455329048) in https://github.com/tidepool-org/LoopKit/pull/142

**Review by rickpasetto — APPROVED** [review]
LGTM 📎

**Pull request merged**